receipt.go

```diff
@@ -0,0 +1,30 @@
+package omise
+
+import "time"
+
+// Receipt represents Omise's receipt object.
+// See https://www.omise.co/receipts-api for more information.
```

Comment:
Sorry I also put the wrong URL in the comment. It should be `https://www.omise.co/receipt-api`